content/v2/domains.markdown

```diff
@@ -9,9 +9,11 @@ excerpt: This page documents the DNSimple domains API v2.
 {:toc}
 
 
-## List domains {#list}
+## List domains {#listDomains}
```

Comment:
I see you changed this, but only this instance in each file. Are you planning on changing the other instances of this in another PR (i.e. change from #create to #createDomain)?

Reply:
Eventually, yes. But probably not part of this PR. This is a cleanup that will have to be made, we haven't committed resources to it yet.

I'm slowly making the changes as long as I'm working on any affected endpoint.